internal/github/github.go

```diff
@@ -53,5 +54,13 @@ func (c *Client) executeRequest(req *http.Request) (release, error) {
 }
 
 func isUpdateRequired(current, remote string) bool {
-	return semver.Compare(current, remote) < 0
+	currentV := current
```

Comment:
I think it would be nice to have a comment here to indicate what should be like for `current` and `remote` 🤔 . It's a bit confusing to think what are they comparing

Reply:
@tianfeng92 I updated using `currentVersion` and `githubVersion`, is that better for you ?

Reply:
Cool! 👍